libraries/api-documenter/run.cmd

```diff
@@ -0,0 +1,3 @@
+@ECHO OFF
+@SETLOCAL
+node "%~dp0\lib\index" %*
```

Comment:
I create a command line eventually, once the tool is implemented.  The "run.cmd" is used for debugging, just like api-extractor/run.cmd

---
In reply to: [134341762](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134341762) [](ancestors = 134341762)